shared/telemetry.ts

```diff
@@ -7,13 +8,21 @@ import * as sourcegraph from 'sourcegraph'
  */
 export class TelemetryEmitter {
     private languageID: string
+    private repoID: string
     private started: number
     private enabled: boolean
     private emitted = new Set<string>()
 
-    constructor(languageID: string, enabled = true) {
+    /**
+     *
```

Comment:
Stupid nit, could you put in a dumb sentence here? (Also the repoID docstring isn't complete).